package.json

```diff
@@ -15,10 +15,11 @@
   "scripts": {
     "lint": "xo generator.js",
     "generate-example": "npm run example-generate",
-    "example-generate": "shx rm -rf tram-one-example && node generator.js tram-one-example",
+    "example-generate": "rm -rf tram-one-example && node generator.js tram-one-example",
```

Comment:
removing shx, as it was having issues on windows deleting a folder. People recommended using other tools like rimraf, either way, until we find a more consistent tool, I'm not actually that passionate about supporting windows systems (that aren't already using some bash terminal).